pkg/remotewrite/client.go

```diff
@@ -93,7 +95,7 @@ func (r *Client) Ingest(ctx context.Context, in *ingestion.IngestInput) error {
 	if !(res.StatusCode >= 200 && res.StatusCode < 300) {
 		// read all the response body
 		respBody, _ := ioutil.ReadAll(res.Body)
-		return fmt.Errorf("%w: %v", ErrNotOkResponse, fmt.Errorf("status code: '%d'. body: '%s'", res.StatusCode, respBody))
+		return fmt.Errorf("%w: %v", ErrNotOkResponse, fmt.Errorf("status code: '%d'. body: '%s'. url: '%s'", res.StatusCode, respBody, r.url))
```

Comment:
Let's use [`r.URL.Redacted()`](https://pkg.go.dev/net/url#URL.Redacted) to not leak basic auth password.

https://go.dev/play/p/2XzEbHtX-OP

Reply:
done, thanks! @eh-am